src/Design/03_InitialPoC_trans.purs

```diff
@@ -1,56 +1,52 @@
 module PSPS.Design.InitialPoC.Trans where
 
-import PSPS (class Bind, class Eval, App, kind Expr, kind ModuleName)
+import FFI.PSPS.Design.InitialPoC.Trans (class FFI_PSPS_Design_InitialPoC_Trans_Eval, Foreign'intAdd, Foreign'intMul)
```

Comment:
When auto-generating, we'll need to create stub FFI Files that have data types for all imported functions as well as their own eval context. Here, since `intAdd` and `intMul` are imported from the FFI, we create a file with `class FFI_PSPS_Design_InitialPoC_Trans_Eval, Foreign'intAdd, Foreign'intMul` and import all 3.
Library authors will have to fill these in, but the advantage is that if the functions are not used, the program will still execute. In this case, the values for `intAdd` and `intMul` are filled in by hand. We should probably do that with the whole prelude.